provider/pkg/gen/schema.go

```diff
@@ -656,7 +654,6 @@ Use the navigation below to see detailed documentation for each of the supported
 	})
 
 	requires := map[string]string{
-		"pulumi":   ">=3.109.0,<4.0.0",
```

Comment:
Was 109.0 highly intentional here? Are we still OK here?

Reply:
It was intentional to make use of a change in `>=3.109.0` of the core SDK, but this change LGTM as it's now `>=3.134.1` as desired.